src/locales/en.json

```diff
@@ -117,6 +117,7 @@
     "host_provider": "Host",
     "insurance": "Insurance",
     "isp": "ISP",
+    "online_services": "Services en ligne",
```

Comment:
I don't see this new one in the `/config/categories.json` it must be added to be completely handled by the Store.